templates/providerstoragecluster.go

```diff
@@ -19,8 +19,8 @@ package templates
 import (
```

Comment:
We will add this when it will be required. Remove it for now

Reply:
Removed in the latest commit

Reply:
It is still there. along with the convergedstoragecluster